src/utilities.js

```diff
@@ -0,0 +1,75 @@
+import dayjs from "dayjs";
+import numeral from "numeral";
+
+export const cleanMovieData = (movie) => {
```

Comment:
Much needed file. Better to cleanse the data from the api call in this file vs. doing it in our components.